test/template_literal.spec.js

```diff
@@ -4,36 +4,37 @@
  **/
 /* eslint-disable no-template-curly-in-string */
 /* eslint-disable no-multi-str */
-const { rewriteAndExpectNoTransformation, rewriteAndExpect } = require('./util')
+const { rewriteAst, rewriteAndExpectNoTransformation, rewriteAndExpect } = require('./util')
```

Comment:
Most of the changes in this file are only tabs, I'll add comments in changed points